codespell_lib/data/dictionary.txt

```diff
@@ -3029,12 +3029,14 @@ nkwo->know
 nmae->name
 nodel->model
 nodels->models
+non->none
```

Comment:
`non-` is standard in British English apparently, I worry this will cause problems

Reply:
can we test if it's followed by a hyphen? In other words if it's `non-` then disregard?

Reply:
Not sure, you could test it.

A quick `git grep` tells me that `scipy` has 33 instances of ` non `. Even the Wikipedia article on [Non-negative matrix factorization](https://en.wikipedia.org/wiki/Non-negative_matrix_factorization) has a `... non negative.`, presumably because of the rule about not hyphenating compound adjectives if they don't precede a noun. So it might even be considered correct in some circles (?).

I suspect this change will create more problems for people than it solves.